sqlmesh/core/model/common.py

```diff
@@ -176,11 +176,10 @@ def single_value_or_tuple(values: t.Sequence) -> exp.Identifier | exp.Tuple:
     )
 
 
-@field_validator_v1_args
 def parse_expression(
     cls: t.Type,
     v: t.Union[t.List[str], t.List[exp.Expression], str, exp.Expression, t.Callable, None],
-    values: t.Dict[str, t.Any],
+    info: t.Optional[ValidationInfo] = None,
```

Comment:
Why is this optional?

Reply:
Made it optional to allow the tests to use `parse_expression`. Did a quick check to see if I can mock `info` but wasn't sure how to do it, can revisit if needed.

The `parse_properties` validator is also invoked in `model/kind.py` without an info argument, so it being optional makes more sense there from a functionality standpoint:

```python
            tuple_exp = parse_properties(cls, v)
```

Reply:
So can we not make it optional?

Reply:
I took another look, I'm not sure how this can be done because `ValidationInfo` is a protocol, so I can't instantiate a dummy instance for it in the test, or in the above `parse_properties` call.

One alternative is to extract the logic into a separate helper function that expects a dialect argument, and then call that instead so we can bypass the validation info argument altogether in other call sites. What do you think?

Reply:
ok, never mind